pymc/gp/hsgp_approx.py

```diff
@@ -301,9 +301,14 @@ def prior_linearized(self, Xs: TensorLike):
                 # as m_star.
                 beta = pm.Normal("beta", size=gp._m_star)
 
-                # The (non-centered) GP approximation is given by
+                # The (non-centered) GP approximation is given by:
                 f = pm.Deterministic("f", phi @ (beta * sqrt_psd))
 
+                # The centered approximation can be more efficient when
+                # the GP is stronger than the noise
+                # beta = pm.Normal("beta", sigma=sqrt_psd, size=gp._m_star)
+                # f = pm.Deterministic("f", phi @ beta)
```

Comment:
Out of curiosity: is this something empirical, or is there a general statement about it? :)